docs/powerpoint/tagging-presentations-slides-shapes.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Use custom tags on presentations, slides, and shapes in PowerPoint
+description: 'Learn how to use tags for custom metadata about presentations, slides, and shapes.'
+ms.date: 04/08/2021
+localization_priority: Normal
+---
+
+# Use custom tags for presentations, slides, and shapes in PowerPoint
+
+An add-in can attach custom metadata, in the form of key-value pairs, called "tags", to presentations, specific slides, and specific shapes on a slide.
+
+> [!IMPORTANT]
+> The APIs for tags are in preview. Please experiment with them in a development or testing environment but don't add them to a production add-in.
+
+There are two main scenarios for using tags:
+
+- When applied to a slide or a shape, a tag enables the object to be categorized for batch processing. For example, suppose a presentation has some slides that should be included in presentations to the East region but not the West region. Similarly, there are alternative slides that should be shown only to the West. Your add-in can create a tag with the key `REGION` and the value `East` and apply it to the slides that should only be used in the East. The tag's value is set to `West` for the slides that should only be shown to the West region. Just before a presentation to the East, a button in the add-in runs code that loops through all the slides checking the value of the `REGION` tag. Slides where the region is `West` are deleted. The user then closes the add-in and starts the slide show.
+- When applied to a presentation, a tag is effectively a custom property in the presentation document (similar to a [CustomProperty](/javascript/api/word/word.customproperty) in Word).
+
+## Tagging slides and shapes
+
+A tag is a key-value pair, where the value is always of type `string` and is represented by a [Tag](/javascript/api/powerpoint/powerpoint.tag) object. Each type of parent object, [Presentation](/javascript/api/powerpoint/powerpoint.presentation), [Slide](/javascript/api/powerpoint/powerpoint.slide), and [Shape](/javascript/api/powerpoint/powerpoint.shape) object has a `tags` property of type [TagsCollection](/javascript/api/powerpoint/powerpoint.tagcollection).
+
+### Adding, updating, and deleting tags
+
+To add a tag to an object, call the [TagCollection.add](/javascript/api/powerpoint/powerpoint.tagcollection#add_key__value_) method of the parent object's `tags` property. The following is an example of adding a two tags to the first slide of a presentation. About this code, note:
+
+- The first parameter of the `add` method is the key.
```

Comment:
To reiterate, in case someone skips to this "Adding" section without reading the preceding paragraph. 
```suggestion
- The first parameter of the `add` method is the key in the key-value pair. 
```